src/NLU.DevOps.ModelPerformance/NLUAccuracy.cs

```diff
@@ -14,128 +15,182 @@ namespace NLU.DevOps.ModelPerformance
     /// </summary>
     public static class NLUAccuracy
     {
-        /// <summary>
-        /// Calculates Precision, Recall and F1 Score
-        /// </summary>
-        /// <param name="cm">confusion matrix metrics</param>
-        /// <param name="roundingplace"> Specifies the number of digits after the decimal point</param>
-        /// <returns>List that contains the results calculated</returns>
-        public static List<double> CalcAccuracy(this ConfusionMatrix cm, int roundingplace = 4)
-        {
-            var metrics = new List<double>
-            {
-                Math.Round(cm.Precision(), roundingplace),
-                Math.Round(cm.Recall(), roundingplace),
-                Math.Round(cm.F1(), roundingplace)
-            };
-            return metrics;
-        }
-
         /// <summary>
         /// Prints to the console the intents, entities performance results
         /// and a confusion table for intents
         /// </summary>
         /// <param name="compareResults"> The comparison results for intents and entities</param>
-        public static void PrintResults(this NLUCompareResults compareResults)
+        /// <param name="baseline">The baseline results the results are benchmarked against.</param>
+        public static void PrintResults(this NLUCompareResults compareResults, NLUStatistics baseline)
```

Comment:
>PrintResults [](start = 27, length = 12)

do you have a screenshot examples of results?

Reply:
![image](https://user-images.githubusercontent.com/1106239/76335606-30830f00-62cb-11ea-9e75-23237244a300.png)

Reply:
Inline.